src/libcrypto.c

```diff
@@ -55,7 +55,7 @@
 #include <openssl/des.h>
 #endif
 
-#ifdef HAVE_OPENSSL_MODES_H
+#if defined(HAVE_OPENSSL_MODES_H) || defined(HAVE_OPENSSL_CRYPTO_CTR128_ENCRYPT)
```

Comment:
This was causing a double include on my machine since both defines were set, so here we do a check.